recipes/r-class/bld.bat

```diff
@@ -0,0 +1,8 @@
+"%R%" CMD INSTALL --build .
+if errorlevel 1 exit 1
+
+@rem Add more build steps here, if they are necessary.
+
+@rem See
+@rem http://docs.continuum.io/conda/build.html
+@rem for a list of environment variables that are set during the build process.
```

Comment:
We can remove the comments, but I don't want the build code in the meta.yaml. what I like about conda is the separation of build code and metadata.